frb_example/pure_dart/rust/src/bridge_generated.rs

```diff
@@ -726,11 +726,14 @@ fn wire_handle_stream_sink_at_3_impl(
 fn wire_get_sum_struct_impl(port_: MessagePort) {
     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
         WrapInfo {
-            debug_name: "get_sum_struct",
+            debug_name: "multiply_by_ten",
```

Comment:
nit: btw why `wire_get_sum_struct_impl` has debug name of `multiply_by_ten`?

Reply:
Probably needs regenerating.